final-project/core/backend/app/routers/groups.py

```
@@ -0,0 +1,195 @@
+    # Calculate offset
+    offset = (page - 1) * ITEMS_PER_PAGE
+    
+    # Get unified list of words and kanji with their stats
+    words_query = (
+        select(
+            Word.id,
+            Word.word_level.label("level"),
+            Word.japanese.label("name"),
+            literal('word').label("item_type"),
+            func.count(ReviewItem.id).label("total_reviews"),
+            func.sum(case((ReviewItem.correct == True, 1), else_=0)).label("correct_reviews")
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if ReviewItem.correct:` for truth checks

Replace with `ReviewItem.correct`

(E712)